docker-compose.yml

```diff
@@ -25,4 +25,16 @@ services:
     env_file: .env
     depends_on:
       - db
-      - redis
\ No newline at end of file
+      - redis
+
+  react:
+    build:
+      context: .
+      dockerfile: Dockerfile-frontend
+    stdin_open: true
+    command: npm start
+    env_file: .env
+    ports:
+      - "3000:3000"
+    volumes:
+      - ./frontend:/frontend
```

Comment:
this will have an issue of creating a node_modules folder on your hd which you won't have permissions for